### PR TITLE
Remove tota11y

### PR DESCRIPTION
**Pull request by SeanMcP**
## Summary:
This PR removes tota11y 🪦

Issue: FEI-4773

## Test plan:
- Page looks a-okay

<details>
<summary>Screenshot</summary>

preview

</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -698,6 +669,7 @@ <h2 class="page-title">Open Source</h2>
             <div class="activity"></div>
         </div>
 </div>
+        <div class="card"></div>
```
Comment (SeanMcP) [inline index.html:672]: This does render an empty card at the end of the row. Might be nice to not have that hanging around, but I don't know if it is a show stopper.

File: data.js [hunk 2]
```diff
@@ -28,12 +28,6 @@ var entries = [
         language: "JavaScript",
         projectLogo: "./img/perseus.png",
         description: "Khan Academy's exercise question editor and renderer."
-    }, {
-        projectSlug: "tota11y",
-        projectName: "Tota11y",
-        language: "JavaScript",
-        projectLogo: "./img/tota11y.png",
```
Comment (jeresig) [inline data.js:35]: Could you delete this file, as well?

**Review by jeresig — APPROVED** [review]
Thanks for cleaning this up!

File: data.js (hunk 2 above)
Comment (SeanMcP) [reply data.js:35]: Good call! Done in 0415726

File: index.html (hunk 1 above)
Comment (somewhatabstract) [reply index.html:672]: I suspect we will be adding more stuff here (and removing others). For example, Wonder Stuff isn't listed yet.

**Review by somewhatabstract — COMMENTED** [review]
LGTM - thanks!

**Pull request merged**